database/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres
```

Comment:
By using an offfcial image, that mean we will have update from maintainer of the repos of docker-hub.
We can have security update from that, and depends on their roadmap. 
We can also prevent critical vulnerability with our Dockerfile based on the official and fix vulnerability that are fixable.
I think we can validate this PR, but more we fix vulnerability discovered as soon as possible, less we'll have work later.  Not sure if we can maintain it with official image. 
Also, I don't really know the vector of attack.
This official image has critical vulnerability fixable:
**CVE-2023-24540 Detail
Description**
`Not all valid JavaScript whitespace characters are considered to be whitespace. Templates containing whitespace characters outside of the character set "\t\n\f\r\u0020\u2028\u2029" in JavaScript contexts that also contain actions may not be properly sanitized during execution.` 
 **weakness enumeration:** (source http://cwe.mitre.org/data/definitions/74.html)
`CWE-74: Improper Neutralization of Special Elements in Output Used by a Downstream Component ('Injection')`

Reply:
We can keep this in mind. 
In the scope of this project I see no problem using official images unless they have major vulnerabilities/glitches.
If you think we need to address this, please open an issue @apayetRun so we can fix it with another PR later on.